documentation/book/security.adoc

```diff
@@ -1,61 +1,75 @@
-* `<cluster-name>-kafka-clients-ca`: contains the private and public keys, so the self-signed certificate, used for signing broker certificates used for communicating with clients. It is specific for each deployed Kafka cluster as specified in the <cluster-name> prefix.
-* `<cluster-name>-kafka-cert`: contains only the public keys, so the self-signed certificate, used for signing broker certificates used for communicating with clients. This Secret should be used by the final Kafka cluster user in order to extract this certificate to put into the the client's truststores for verifying broker identities (server authentication).
-* `<cluster-name>-kafka-brokers-internal`: contains all the brokers private and public keys (certificates signed with "internal-ca") used for interbroker communication.
-* `<cluster-name>-kafka-brokers-clients`: contains all the brokers private and public keys (certificates signed with specific cluster "clients-ca") used for communicating with clients.
+`<cluster-name>-cluster-ca`::
+Contains the private and public keys of the cluster CA wich is used for signing server certificates for the Kafka and Strimzi components (Kafka brokers, Zookeeper nodes and so on).
+`<cluster-name>-cluster-ca-cert`::
+Contains only the public key of the cluster CA.
+The public key should be used by Kafka clients to verify the identity of the Kafka brokers they are connecting to.
+`<cluster-name>-clients-ca`::
+Contains the private and public keys of the clients CA.
+The clients CA is used for TLS client authentication of Kafka clients when connecting to Kafka brokers.
+`<cluster-name>-clients-ca-cert`::
+Contains only the public key of the client CA.
+The public key is used by the Kafka brokers to verify then identity of Kafka clients when TLS client authentication is used.
+`<cluster-name>-kafka-brokers`::
+Contains all the Kafka broker private and public keys (certificates signed with the cluster CA).
+`<cluster-name>-zookeeper-nodes`::
+Contains all the Zookeeper node private and public keys (certificates signed with the cluster CA).
 
 All the keys are 2048 bits in size and are valid for 365 days from initial generation.
 
 NOTE: "certificates rotation" for generating new ones on their expiration will be supported in future releases.
 
-=== Listeners
+=== Kafka Listeners
 
 The data encryption is provided on two different listeners exposed by each Kafka broker.
 
-Interbroker communication is through the `REPLICATION` listener on port 9091, which is encrypted by default.
+Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
 Encrypted communication with clients is provided through the `CLIENTTLS` listener on port 9093.
 
 NOTE: Un-encrypted communication with clients is still possible through the `CLIENT` listener on port 9092.
```

Comment:
Please change "Un-encrypted communication with clients is still possible through the `CLIENT` listener on port 9092." to "You can use the `CLIENT` listener on port 9092. for unencrypted communication with clients."